docs/src/hohqmesh_tutorial.md

```diff
@@ -0,0 +1,481 @@
+# Unstructured meshes with HOHQMesh.jl
+
+Trixi.jl supports numerical approximations on unstructured quadrilateral meshes
+with the `UnstructuredMesh2D` mesh type.
+
+The purpose of this tutorial is to demonstrate how one can use the `UnstructuredMesh2D`
+functionality of Trixi.jl. This begins by running and visualizing an available unstructured
+quadrilateral mesh example. Then, the tutorial will demonstrate how one can
+conceptualize a problem with curved boundaries, generate
+a curvilinear mesh using the available software in the Trixi.jl ecosystem,
+and then run a simulation using Trixi.jl on said mesh.
+
+Unstructured quadrilateral meshes can be made
+with the [*High-Order Hex-Quad Mesh (HOHQMesh)*](https://github.com/trixi-framework/HOHQMesh)
```

Comment:
What does the `*` in `(HOHQMesh)*` mean?

Reply:
I will remove this. It is supposed to make it italic but that was the style before I put in a link